publisher/uid2-java-test-site/src/test/java/com/uid2/testsite/TestMainVerticle.java

```diff
@@ -0,0 +1,22 @@
+package com.uid2.testsite;
+
+import io.vertx.core.Vertx;
+import io.vertx.junit5.VertxExtension;
+import io.vertx.junit5.VertxTestContext;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(VertxExtension.class)
+public class TestMainVerticle {
+
+  @BeforeEach
+  void deploy_verticle(Vertx vertx, VertxTestContext testContext) {
```

Comment:
NIT: camel case?

Reply:
vertx generates this, so I'm hesitant to touch it. Not sure why they use this style
https://github.com/search?l=Java&q=deploy_verticle&type=Code